src/layouts/HelpCenterLayout/components/FAQs/FAQs.vue

```diff
@@ -0,0 +1,248 @@
+              <h3>Withdrawing DAO tokens</h3>
+              <div class="show-more-buttons">
+                <span v-if="showFAQs.faq12 == false" class="show-more">+</span>
+                <span v-if="showFAQs.faq12 == true" class="show-less">-</span>
+              </div>
+            </div>
+            <div v-if="showFAQs.faq12" class="qa__contents--content">
+              <faq-withdrawing-dao-tokens />
+            </div>
+          </li>
+          <li>
+            <div v-on:click="openFAQ('faq13')" class="qa__contents--title">
+              <h3>Why don't you make me move my mouse to generate a seed?</h3>
+              <div class="show-more-buttons">
+                <span v-if="showFAQs.faq13 == false" class="show-more">+</span>
+                <span v-if="showFAQs.faq13 == true" class="show-less">-</span>
+              </div>
+            </div>
+            <div v-if="showFAQs.faq13" class="qa__contents--content">
+              <faq-why-dont-you-make-me-move-my-mouse-to-generate-a-seed />
+            </div>
+          </li>
+
+        </ul>
+      </div>
+
+    </div>
+  </div>
+</template>
+
+<script>
+import {
+  BalancesNotLoadingOrShowingOnMyEtherWallet,
+  EthOrTokensSentToOrFromExchangeHaventShownUp,
+  DoesMyEtherWalletSupportBitcoinOrOtherCoins,
+  WhereCanIBuySellTradeExchangeMyEthOrTokens,
+  WhatIsAnIco,
+  HowToBuyEthereumWithUSD,
+  CouldntEverybodyPutInRandomPrivateKeys,
+  WhatHappensIfMyEtherWalletGoesDown,
+  HowDoIInteractWithAMultisigContract,
+  CheckTheCurrentBlockNumberOnMyEtherWallet,
+  WhenIUnlockViaKeystoreFileItFreezesMyBrowser,
+  WithdrawingDAOTokens,
+  WhyDontYouMakeMeMoveMyMouseToGenerateASeed
+} from '@/components/FaqContents'
+
+export default {
+  components: {
+    'faq-balances-not-loading': BalancesNotLoadingOrShowingOnMyEtherWallet,
+    'faq-eth-or-tokens-sent-to': EthOrTokensSentToOrFromExchangeHaventShownUp,
+    'faq-does-myetherwallet-support-bitcoin': DoesMyEtherWalletSupportBitcoinOrOtherCoins,
+    'faq-where-can-i-buy-sell': WhereCanIBuySellTradeExchangeMyEthOrTokens,
+    'faq-what-is-an-ico': WhatIsAnIco,
+    'faq-how-to-buy-ethereum-with-usd': HowToBuyEthereumWithUSD,
+    'faq-couldnt-everybody-put-in-random-private-keys': CouldntEverybodyPutInRandomPrivateKeys,
+    'faq-what-happens-if-my-ether-wallet-goes-down': WhatHappensIfMyEtherWalletGoesDown,
+    'faq-how-do-i-interact-with-a-multisig-contract': HowDoIInteractWithAMultisigContract,
+    'faq-check-the-current-block-number-on-my-ether-wallet': CheckTheCurrentBlockNumberOnMyEtherWallet,
+    'faq-when-i-unlock-via-keystore-file-it-freezes-my-browser': WhenIUnlockViaKeystoreFileItFreezesMyBrowser,
+    'faq-withdrawing-dao-tokens': WithdrawingDAOTokens,
+    'faq-why-dont-you-make-me-move-my-mouse-to-generate-a-seed': WhyDontYouMakeMeMoveMyMouseToGenerateASeed
+  },
+  methods: {
+    openFAQ: function (faqToOpen) {
+      var currentState = this.showFAQs[faqToOpen]
+      this.showFAQs.faq1 = false
```

Comment:
Can't you replace this with a string instead of having multiple booleans? use v-show to check whether the article should be shown or not.